routes/models/och_log_entry_schema.js

```diff
@@ -0,0 +1,30 @@
+import mongoose from 'mongoose';
+const { Schema } = mongoose;
+
+const logEntry = new Schema({
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'logEntry' is assigned a value but never used.